themes/digital.gov/layouts/guides/card-guide.html

```diff
@@ -0,0 +1,14 @@
+<article class="dg-guides-card">
```

Comment:
There's no comment as to what this component is for, where it's used, or how it's used.

Additionally, which fields are required vs optional?

For example the snippet below will always render an image, even if it doesn't exist.

Example:
```html
<div class="dg-guides-card__media">
    {{- partial "core/img-flexible.html" (dict "Site" .Site "src" .Params.Image "small" "false") -}}
  </div>
```

![image](https://github.com/GSA/digitalgov.gov/assets/3385219/750b757c-b643-4ba3-be41-f856d6c5909d)

Reply:
Thanks for pointing this out, I've added documentation to this partial to explain it's function and use.

Added conditionals for both `$summary` and `$image` fields because there is a potential for them to be missing unlike `$title`.